sql/analyzer/indexed_joins.go

```diff
@@ -378,6 +382,83 @@ func convertSemiToInnerJoin(a *Analyzer, m *Memo) error {
 	})
 }
 
+// convertAntiToLeftJoin adds left join alternatives for anti join
```

Comment:
I find simple transforms helpful for reminding me how these work: `ANTI_JOIN(left, right) => PROJECT(left sch) -> FILTER(right attr IS NULL) -> LEFT_JOIN(left, right)`